js/ui/overlay/ui.overlay.js

```diff
@@ -401,6 +402,11 @@ const Overlay = Widget.inherit({
         };
     },
 
+    _initMarkup() {
+        this._renderWrapperAttributes();
+        this.callBase();
```

Comment:
Are u sure that we should render wrapper attributes before calbase?

Reply:
nope, replaced to bottom